climada/hazard/tc_surge_bathtub.py

```diff
@@ -29,7 +29,7 @@
 import scipy.sparse as sp
 
 from climada.hazard.base import Hazard
-import climada.util.coordinates as coord_util
+import climada.util.coordinates as co
```

Comment:
Since it also affected trop_cyclone.py, I used this PR to unify the shorthand for climada.util.coordinates everywhere in the whole project. The shorthand co is taken from the base modules in the hazard and exposures subpackages.